### ISSUE TITLE
Non-edittable cells 

### ISSUE THREAD

**Issue by eclectir** [jupyter/help#80]
There are several documentation and edu scenarios where I'd want some cells (markdown and code) in my notebook to be non-editable. Would be awesome to have an option to mark cells non-editable.

**Comment by jhamrick** [comment]
Yes, I definitely agree! We have an issue open for this on the notebook repo: https://github.com/jupyter/notebook/issues/1402
